src/pages/ReimbursementAccount/BeneficialOwnersStep.js

```diff
@@ -110,6 +121,25 @@ class BeneficialOwnersStep extends React.Component {
         () => setupWithdrawalAccount({...this.state}));
     }
 
+    /**
+    *
+    * @param {Object} value
+    */
+    debouncedUpdateReimbursementAccountDraft(value) {
+        updateReimbursementAccountDraft(value);
+    }
+
+    /**
+    *
+    * @param {Object} fieldName
+    */
+    toggleCheckbox(fieldName) {
+        this.setState((prevState) => {
+            this.debouncedUpdateReimbursementAccountDraft({[fieldName]: !prevState[fieldName]});
+            return {[fieldName]: !prevState[fieldName]};
```

Comment:
Looks like this is the same as above, can you save it to a variable instead of repeating?